rest/blip_sync.go

```diff
@@ -95,6 +96,11 @@ func (h *handler) handleBLIPSync() error {
 		blip.CompressionLevel = *c
 	}
 
+	var useDeltas bool
+	if enabled := h.server.GetDatabaseConfig(h.db.Name).DeltaSync.Enable; enabled != nil {
+		useDeltas = *enabled
+	}
```

Comment:
Should be disabled for a given blip context until client explicitly opts in (and SG supports), as pre-Iridium clients won't be sending the flag.